src/org/usfirst/frc4904/robot/commands/MotorIdleGroup.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Idles motors of a subsystem.
+ * Used as a default command
+ *
+ */
+public class MotorIdleGroup extends CommandGroup{
+
+	/**
+	 * Run MotorIdle (from WPILib) in parallel on every motor
+	 * from a list of motors with unrestricted size.
+	 *
+	 * @param subsystem
+	 * @param motors
+	 */
+	public MotorIdleGroup(Subsystem subsystem, Motor... motors) {
+		super("IdleMotors");
```

Comment:
People should be able to name the idling motors for debugging, so can you make the name a parameter with another constructor that extends it with this as the name.

Reply:
it will be an array then because you can't use the ... syntax twice in a constructor

Reply:
@billwpierce you can name motors then pass it into the command

Reply:
do you want it to LogKitten the motor names?

Reply:
that's not the problem, it's that you want people to know which idle motors are idling, otherwise the list of running commands in the logs is going to be long and repetitive